config/common/group_vars/k8s_cluster/ck8s-k8s-cluster.yaml

```diff
@@ -751,3 +751,10 @@ local_volume_provisioner_storage_classes: # Installation via kubespray has no su
 
 local_volume_provisioner_nodelabels: []
 local_volume_provisioner_tolerations: []
+
+ntp_filter_interface: true
+# Specify the interfaces
+# Only takes effect when ntp_filter_interface is true
+ntp_interfaces:
```

Comment:
Do these depend on an updated kubespray submodule?

Reply:
Yes, it does.

This config is a feature in an upcoming kubespray release 2.25. See [here](https://github.com/kubernetes-sigs/kubespray/issues/11126#:~:text=Added%20a%20config%20option%20to%20filter%20ntp%20interfaces%20(ntp%3A%20add%20config%20to%20filter%20and%20set%20ntp%20interfaces%C2%A0%2311066%2C%20%40Pavan%2DGunda))